Networking/Networking/Mapper/SuccessDataResultMapper.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+
+/// Mapper: Success Result Wrapped in `data` Key
```

Comment:
It's good to know that this is different from the `SuccessResult` model, because of the `data` key. 💯